### ISSUE TITLE
Cleaning out the issue tracker

### ISSUE THREAD

**Issue by jonnenauha** [realXtend/WebTundra#81]
@antont @cadaver @cvetan5 should we just close all the issues not directly related to dev2 as `wontfix`.

Would really like to get a fresh start with the issues. Some might transfer over but I think they need to be evaluated after the first release.

**Comment by antont** [comment]
yep - was reading the issues with this in mind the other day and almost started closing already, but figured it is cleaner perhaps after the 2.0 release (or closer to it) if someone uses 1.0 / master now and encounters issues with it, that the known ones show in the tracker.

**Comment by cadaver** [comment]
Yes, fine with this, once 2.0 becomes the current release.

**Comment by antont** [comment]
first pass of closing obviously old irrelevant to current ones done.
